### ISSUE TITLE
jlqml assumes QML is in scope

### ISSUE THREAD

**Issue by bramtayl** [JuliaGraphics/QML.jl#95]
I'm working on a QML application at Justly.jl and I'm getting this error if I run without first loading QML:

```
ERROR: Could not find module QML when looking up function get_julia_call
Stacktrace:
 [1] exec at /home/brandon/.julia/packages/CxxWrap/ZOkSN/src/CxxWrap.jl:590 [inlined]
 [2] justly_interactive(; wave::AudioSchedules.SawTooth{7}, make_envelope::Function, initial_key::Unitful.Quantity{Int64,𝐓^-1,Unitful.FreeUnits{(Hz,),𝐓^-1,nothing}}, beat_duration::Unitful.Quantity{Int64,𝐓,Unitful.FreeUnits{(s,),𝐓,nothing}}, test::Bool) at /home/brandon/.julia/dev/Justly/src/Justly.jl:220
 [3] justly_interactive() at /home/brandon/.julia/dev/Justly/src/Justly.jl:179
 [4] top-level scope at REPL[8]:1
 [5] include_string(::Function, ::Module, ::String, ::String) at ./loading.jl:1088
```

I suspect it's coming from a hygeine issue with the Base `@cfunction` macro but I'm not 100% sure.

**Comment by barche** [comment]
This is actually coming from here: 

https://github.com/barche/jlqml/blob/1e2fb83a7805b7ee7048b6cb1d22fa4bcf59095b/julia_function.cpp#L22

That line assumes there is a module "QML" in scope, to fix this I will have to make changes to jlqml.

**Comment by utkarshraj779** [comment]
Hlw @barche I am Utkarsh Kumar from (IIT KHARAGPUR in 3rd year pursuing BTech) and i want to contribute to GSOC 2023 in this project based on the C++ please give me some guidance where i should i start working.

**Comment by barche** [comment]
Hi, for this particular issue the way to solve it would be to add a global pointer to the QML module, as is done in `libcxxwrap-julia`: https://github.com/JuliaInterop/libcxxwrap-julia/blob/9fa283de6066ac4c2a991d8d715152f2933bb67a/include/jlcxx/module.hpp#L139

**Comment by ufechner7** [comment]
Is there an example to reproduce the problem?

**Comment by barche** [comment]
This is actually already implemented, v0.8.1 contains the last missed case of where the QML module was referenced explicitly.
